_includes/tutorials/multi-joins/ksql/markup/dev/create-inputs-orders.adoc

```diff
@@ -0,0 +1,6 @@
+Next we create a stream containing orders submitted to our online store, also formatted in JSON.
```

Comment:
It may be worth pointing out that this stream contains fields shared by the other two previously created tables.

Reply:
You mean fields that are related?